src/components/PostCard/TopicCard.tsx

```diff
@@ -12,46 +12,70 @@ import {
 } from "react-native";
 import { CardContent } from "./CardContent";
 import { TopicTab } from "./TopicTab";
+import {
+  returnTopicProps as ReturnTopicProps,
+  returnUserProps as ReturnUserProps,
+} from "@src/@types/returnProps";
+import timeTransfer from "@src/utils/timeTransfer";
+import { useQuery } from "@tanstack/react-query/build/lib/useQuery";
+import { getContents } from "@src/services/viewService/getContent";
+import { getUserInfo } from "@src/services/viewService/getUserInfo";
 
 const windowWidth = Dimensions.get("window").width;
 const windowHeight = Dimensions.get("window").height;
 
-interface TopicCardProps {
-  children?: React.ReactNode;
-  title?: string;
-  username: string;
-  userAvatar: string;
-  time: string;
-  Tab?: string[];
-}
-
 export default function TopicCard({
-  children,
+  category,
+  cid,
+  createTime,
+  creator,
+  id,
+  images,
+  tags,
   title,
-  userAvatar,
-  username,
-  time,
-}: TopicCardProps) {
+  updateTime,
+}: ReturnTopicProps) {
```

Comment:
Never destruct an unused variable to suppress eslint warning.
I allow `_` as a legal name for the unused variable. Use it when necessary.